Description: add Makefile target dependency for manpages
Author: Lars Kruse <devel@sumpfralle.de>
Applied-Upstream: c3cce3215b48cbc3ddb2a2c75543d8c964309196
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -336,7 +336,7 @@ build-doc-stamp:
 
 build-man: build-man-stamp Makefile Makefile.config
 
-build-man-stamp:
+build-man-stamp: infiles
 	touch build-man-stamp
 	mkdir -p build/doc
 	for f in $(MAN8); do \
